kornia/geometry/epipolar/fundamental.py

```diff
@@ -70,9 +70,210 @@ def normalize_transformation(M: Tensor, eps: float = 1e-8) -> Tensor:
     return torch.where(norm_val.abs() > eps, M / (norm_val + eps), M)
 
 
-def find_fundamental(
-    points1: torch.Tensor, points2: torch.Tensor, weights: Optional[torch.Tensor] = None
-) -> torch.Tensor:
+# Reference : https://github.com/opencv/opencv/blob/4.x/modules/calib3d/src/polynom_solver.cpp
+def solve_quadratic(a: Tensor, b: Tensor, c: Tensor) -> Tensor:
+    delta = b * b - 4 * a * c
+
+    if delta < 0:
+        return torch.tensor(0, device=a.device, dtype=a.dtype)
+
+    inv_2a = 0.5 / a
+
+    if delta == 0:
+        x1 = -b * inv_2a
+        x2 = x1
+        return x1
+
+    sqrt_delta = torch.sqrt(delta)
+    x1 = (-b + sqrt_delta) * inv_2a
+    x2 = (-b - sqrt_delta) * inv_2a
+
+    return torch.cat([x2, x1])
+
+
+def solve_cubic(a: Tensor, b: Tensor, c: Tensor, d: Tensor) -> Tensor:
+    _PI = torch.tensor(3.141592653589793, device=a.device, dtype=a.dtype)
```

Comment:
```suggestion
    _PI = torch.tensor(math.pi, device=a.device, dtype=a.dtype)
```

Reply:
we also have `from kornia.constants import pi `-- https://github.com/kornia/kornia/blob/master/kornia/constants.py#L10